specification.markdown

```diff
@@ -1155,6 +1155,32 @@ The value represents data that **MAY** be stored within the EBML Element.
 
 The `value` attribute is **REQUIRED**.
 
+#### minver
+
+Within an EBML Schema, the XPath of the `@value` attribute is
+`/EBMLSchema/element/restriction/enum/@minver`.
+
+The `minver` (minimum version) attribute stores a nonnegative integer that
+represents the first version of the docType to support the enum value. The value
+**MUST** be greater than or equal to the `minver` attribute value of the EBML Element it belongs to.
+
+The `minver` attribute is **OPTIONAL**. If the `minver`
+attribute is not present, then the enum value has the same value as the `minver` attribute of the
+EBML Element it belongs to.
+
+#### maxver
+
+Within an EBML Schema, the XPath of the `@value` attribute is
+`/EBMLSchema/element/restriction/enum/@maxver`.
+
+The `maxver` (maximum version) attribute stores a nonnegative integer that
+represents the last or most recent version of the docType to support
+the enum value. `maxver` **MUST** be greater than or equal to the `minver` attribute.
+
+The `maxver` attribute is **OPTIONAL**. If the `maxver` attribute is
+not present, then the enum value has a maximum version equal to the value
+stored in the `version` attribute of `<EBMLSchema>`.
```

Comment:
Same concern about the wording as above. Maybe:

> If the `maxver` attribute is not present, then the enum value's maximum version is the value stored in the `version` attribute of `<EBMLSchema>`.

Reply:
OK